cmake/sysbuild/suit.cmake

```diff
@@ -176,12 +176,20 @@ function(suit_create_package)
   foreach(image ${IMAGES})
     sysbuild_get(BINARY_DIR IMAGE ${image} VAR APPLICATION_BINARY_DIR CACHE)
     sysbuild_get(BINARY_FILE IMAGE ${image} VAR CONFIG_KERNEL_BIN_NAME KCONFIG)
+    sysbuild_get(target IMAGE ${image} VAR CONFIG_SUIT_ENVELOPE_TARGET KCONFIG)
```

Comment:
It is necessary to add `unset` here, otherwise the parameters to suit-generator are wrong.